packages/app/package.json

```diff
@@ -31,6 +31,7 @@
     "esbuild": "^0.14.47",
     "fuse.js": "^6.6.2",
     "immer": "^9.0.21",
+    "jotai": "^2.5.1",
```

Comment:
ah it finally made its way into the project... @joelseq I think you'll enjoy this 😂